homeassistant/components/notify/tibber.py

```diff
@@ -0,0 +1,37 @@
+"""
+Pushbullet platform for notify component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.tibber/
+"""
+import asyncio
+import logging
+
+from homeassistant.components.notify import (
+    ATTR_TITLE, ATTR_TITLE_DEFAULT, BaseNotificationService)
+from homeassistant.components.tibber import DOMAIN
```

Comment:
Please import the tibber domain as another name. This platform belongs to the notify domain.